cookbooks/bcpc-hadoop/attributes/monitoring.rb

```diff
@@ -0,0 +1,5 @@
+default["bcpc"]["hadoop"]["zabbix"]["history_days"] = 1
+default["bcpc"]["hadoop"]["zabbix"]["trend_days"] = 15
+default["bcpc"]["hadoop"]["zabbix"]["cron_check_time"] = 240
```

Comment:
It looks like `cron_check` is being pulled out? (For what it is worth, I believe cron_check was the verify that the Zabbix sender had not died; I'm not sure why it got commented out though -- or was not enabled?)